modules/all/all.go

```diff
@@ -15,4 +15,5 @@ import (
 	_ "github.com/netdata/go.d.plugin/modules/solr"
 	_ "github.com/netdata/go.d.plugin/modules/springboot2"
 	_ "github.com/netdata/go.d.plugin/modules/weblog"
+	// Oracle DB should be configured with specific dsn.
```

Comment:
Should be imported in `all` package. It adds module to the registry.

Reply:
✅